problems/5.loops.js

```diff
@@ -3,16 +3,27 @@
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
 function evenNumbers() {
-
+  let arr = []
+  let num = 0
+  while (num > 0) {
```

Comment:
On the line before you set num to be 0. This loop will never run. We need to accept parameters into our functions